spec/model/postgres/postgres_server_spec.rb

```diff
@@ -519,4 +519,27 @@ def create_standby_resource(suffix)
       expect { postgres_server.refresh_walg_credentials }.not_to raise_error
     end
   end
+
+  describe "#attach_s3_policy_if_needed" do
+    before do
+      skip unless ENV["CLOVER_FREEZE"] != "1"
```

Comment:
Still uses `skip` instead of wrapping with `if`.

Reply:
should I be wrapping the entire `describe` here?